### PR TITLE
Removing misnamed channel

### PR DESCRIPTION
**Pull request by robmoore**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@robmoore Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-integration&pullRequestId=2101)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-integration&pullRequestId=2101) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by artembilan** [comment]
@robmoore ,

please, sing the CLA. We can't review and merge without that.

Thanks

**Comment by robmoore** [comment]
@artembilan Done.

**Comment by pivotal-issuemaster** [comment]
@robmoore Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-integration&pullRequestId=2101)!

**Review by artembilan — APPROVED** [review]

File: src/reference/asciidoc/logging-adapter.adoc [hunk 1]
```diff
@@ -72,8 +67,8 @@ public class LoggingJavaApplication {
         return adapter;
     }
 
-    @MessagingGateway(defaultRequestChannel = "logChannel")
-    public interface MyGateway {
+   @MessagingGateway(defaultRequestChannel = "logChannel") // logChannel is created above via @ServiceActivator
```
Comment (artembilan) [inline src/reference/asciidoc/logging-adapter.adoc]: We don't need this comment here.
It's not responsibility of this chapter to describe that behavior.

Comment (artembilan) [inline src/reference/asciidoc/logging-adapter.adoc]: The indent must be 4 spaces, so reverting during merge...

**Review by artembilan — APPROVED** [review]
Merging with some polishing...

**Comment by artembilan** [comment]
Merged as https://github.com/spring-projects/spring-integration/commit/970854fd6fb87e01bc310a2c4ee83d8bdce53467.

@robmoore ,

Thank you for the contribution.
For the future, please, consider to follow the [Contribution Guideline](https://github.com/spring-projects/spring-integration/blob/master/CONTRIBUTING.adoc), especially don't create Pull Requests from the `master`. Right now, after merging you need to `reset` your local master to the `upstream` one to keep you copy sync.
Looking forward for more of course!

**Comment by robmoore** [comment]
@artembilan Thanks for this. Clearly my misunderstanding of the process resulted in more work for you than if you'd done it yourself to being with. I appreciate your patience and understanding and will be diligent to do better in future.

**Comment by artembilan** [comment]
@robmoore,
Well, you've already done good job finding that bug and providing the fix. Everything rest is really our responsibility. And my comment is more about convenience and saving some time during this contribution process. You don't do something wrong, that's it. 

Frankly that is apologies from other side to make those bugs and make you to fix it 😀

**Pull request closed without merging**